lerobot/templates/visualize_dataset_template.html

```diff
@@ -100,6 +100,14 @@ <h1 class="text-xl font-bold mt-4 font-mono">
             {% endfor %}
         </div>
 
+        <!-- Language instruction -->
+        {% if videos_info[0].language_instruction %}
+        <div class="text-sm font-bold mt-2">
+            Language Instruction: <em>{{ videos_info[0].language_instruction }}</em>
+        </div>
+        <hr class="my-4 border-gray-600">
```

Comment:
```suggestion
        <p class="font-medium mt-2">
            Language Instruction: <span class="italic">{{ videos_info[0].language_instruction }}</span>
        </p>
```

looks good to me. if I'd suggest a stylistic change, it would be:
1. remove the border
2. change the text to be regular size and sembold rather than small text fully bold

Reply:
this is how the suggested change is lookign

<img width="808" alt="image" src="https://github.com/user-attachments/assets/d5ef96c9-9bb0-48cb-b593-d46668adab79">